sympy/ntheory/factor_.py

```diff
@@ -1304,6 +1304,90 @@ def divisor_count(n, modulus=1):
     return Mul(*[v + 1 for k, v in factorint(n).items() if k > 1])
 
 
+def _udivisors(n):
+    """Helper function for udivisors which generates the unitary divisors."""
+
+    factorpows = [p**e for p, e in factorint(n).items()]
+    for i in range(2**len(factorpows)):
+        d, j, k = 1, i, 0
+        while j:
+            if (j & 1):
+                d *= factorpows[k]
+            j >>= 1
+            k += 1
+        yield d
+
+
+def udivisors(n, generator=False):
+    r"""
+    Return all unitary divisors of n sorted from 1..n by default.
+    If generator is True an unordered generator is returned.
```

Comment:
Formatting nitpick: It's a good idea to enclose Python expressions in docstrings in double backticks (\`\`).
This will cause the HTML generator to use a monospaced font.
Here, it should be \`\`True\`\` so the HTML will render as `True`.